policy/modules/system/userdomain.if

```diff
@@ -648,6 +649,9 @@ template(`userdom_common_user_template',`
 
 	allow $1_t unpriv_userdomain:fd use;
 
+	# for Kanidm, which manages user home dirs with a symlink and a UUID for each user
+	allow $1_t user_home_dir_t:lnk_file read_lnk_file_perms;
```

Comment:
This should be added to `userdom_ro_home_role()` and `userdom_manage_home_role()` instead. Home dirs might have a symlink for many reasons.